src/main/java/uk/gov/justice/digital/exceptions/DomainExecutorException.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.exceptions;
+
+public class DomainExecutorException extends Throwable {
```

Comment:
Extend `RuntimeException` instead. Checked exceptions aren't really necessary here since most of these exceptions get passed up the call stack anyway.